src/assets/css/components/code.css

```diff
@@ -0,0 +1,17 @@
+@layer components {
+
+  *:not(pre) > * > code {
```

Comment:
This is currently used to select inline code and apply different styling from our code blocks. There might be a better solution but this works for now

Replaces `ProseCodeInline.vue`